src/accumulator/mod.rs

```diff
@@ -1,3 +1,67 @@
+//! This module is the core of the library. It contains all the basic data structures to use Utreexo,
+//! such as accumulator, and proofs. Even though the algorithms are equal, you may use different
```

Comment:
Maybe it's better to phrase it this way:

`s/Even though the algorithms are equal, you may use different data structures to represent the accumulator/The different data structures are compatible with each other and you may use different data structures to represent the accumulator./`

And then capitalize `Each` to make it a sentence.